Essentials/src/main/java/com/earth2me/essentials/commands/Commandseen.java

```diff
@@ -27,12 +27,14 @@ public Commandseen() {
 
     @Override
     protected void run(final Server server, final CommandSource sender, final String commandLabel, final String[] args) throws Exception {
-        final boolean showBan = sender.isAuthorized("essentials.seen.banreason", ess);
-        final boolean showIp = sender.isAuthorized("essentials.seen.ip", ess);
-        final boolean showLocation = sender.isAuthorized("essentials.seen.location", ess);
         if (args.length < 1) {
             throw new NotEnoughArgumentsException();
         }
+        final boolean showBan = sender.isAuthorized("essentials.seen.banreason", ess);
+        final boolean showIp = sender.isAuthorized("essentials.seen.ip", ess);
+        final boolean showLocation = sender.isAuthorized("essentials.seen.location", ess);
+        final boolean searchAccounts = commandLabel.contains("alts") && sender.isAuthorized("essentials.seen.alts", ess);
```

Comment:
```suggestion
        final boolean searchAccounts = commandLabel.equalsIgnoreCase("alts") && showIp;
```
Since this is pretty much just a shortcut for ip checks, IP perms is good here

Reply:
I think it's better to leave it this way - it will allow for some kind of helper to use the command without seeing actual IPs of other players.
Also I added `/ealts`. Is it necessary to use String#equalsIgnoreCase there instead? Do I need to remove `/ealts` or just include additional check? I did so following [`/paytoggle`](https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/main/java/com/earth2me/essentials/commands/Commandpaytoggle.java#L22) example